examples/fci-wave-logn/div-integrate/BOUT.inp

```diff
@@ -46,6 +44,3 @@ bndry_par_ydown = parallel_neumann
 [v]
 
 bndry_par_all = parallel_dirichlet
-
-#bndry_par_yup = parallel_dirichlet(+1.0)
-#bndry_par_ydown = parallel_dirichlet(-1.0)
```

Comment:
Not sure why these got deleted - shouldn't the comments still be attached to the `[v]` section?